internal/terraform/rootmodule/types.go

```diff
@@ -29,6 +33,7 @@ type RootModuleManager interface {
 	AddRootModule(dir string) error
 	PathsToWatch() []string
 	RootModuleByPath(path string) (RootModule, error)
+	RootModuleCandidatesByPath(path string) []string
```

Comment:
You could probably just embed the other interface to keep these in sync I think?